packages/vitest/src/node/reporters/blob.ts

```diff
@@ -39,7 +39,7 @@ export class BlobReporter implements Reporter {
       return [project.getName(), [...project.server.moduleGraph.idToModuleMap.keys()]]
     })
 
-    const report = stringify([this.ctx.version, files, errors, moduleKeys] satisfies MergeReport)
+    const report = stringify([this.ctx.version, files, errors, moduleKeys, coverage] satisfies MergeReport)
```

Comment:
Isn't it too big to store it in a json file? Should we use something like https://www.npmjs.com/package/archiver to dump files in the folder and then merge into a single blob file?

Reply:
The coverage report is much smaller than the raw results collected during tests. Here it's remapped and processed fully. 

Example from Vue repo:

```bash
# provider: 'istanbul'
~/Git/vue (main) $ du -sh .vitest-reports/*
4.3M    .vitest-reports/blob-1-4.json
4.5M    .vitest-reports/blob-2-4.json
4.4M    .vitest-reports/blob-3-4.json
4.4M    .vitest-reports/blob-4-4.json

# provider: 'v8'
~/Git/vue (main) $ du -sh .vitest-reports/*
5.0M    .vitest-reports/blob-1-4.json
5.4M    .vitest-reports/blob-2-4.json
5.3M    .vitest-reports/blob-3-4.json
5.3M    .vitest-reports/blob-4-4.json
```

Any other bigger projects that we should try this in?

Reply:
I mean, if there were only two blobs it would've been 10M each which is quite high I feel like

In this json there are also all console logs - if there are multiple testing-library errors, the size might balloon

I am fine with merging this PR and then working on a tar solution